src/tests.rs

```diff
@@ -283,3 +283,123 @@ fn test_user_defined_reranking_model() {
     assert_eq!(results.len(), documents.len());
     assert_eq!(results.first().unwrap().index, 0);
 }
+
+
+#[test]
+fn compare_python_embeddings() {
+    let model = TextEmbedding::try_new(InitOptions {
+        model_name: EmbeddingModel::AllMiniLML6V2,
```

Comment:
Hey @triandco.
Can you please check for `bge-small-en-1.5`?

Reply:
I'd expect this to be a not applicable to all models.
`bge-small-en-v1.5`'s embeddings were in parity.

Reply:
Yes you are right, bge-small-en-1.5 only use pooling_mode_cls_token.

i’ll extend this with a pooling config loading so we can match each model settings.

Reply:
Also, for a per model post process step, if needed, you can refer to https://github.com/Anush008/fastembed-rs/blob/main/src%2Fmodels%2Fsparse.rs

Used like [so](https://github.com/Anush008/fastembed-rs/blob/main/src%2Fsparse_text_embedding.rs#L236).

Reply:
Also, please add yourself to the authors list in the `Cargo.toml` file.

Reply:
Thank you @Anush008, I’ve added the todo to the PR description. I’ll work through it in the next couple of days.

Reply:
🙏

Reply:
Hi @triandco.

TLDR from #102.

@triandco please don't bother looking into the changes in #102. You can continue working on this PR as is.

Will merge this first and then adapt #102 accordingly.